DEVNOTES.md

```diff
@@ -77,6 +77,10 @@ that the `e2e.baseUrl` looks like this:
 ```shell
     baseUrl: 'https://local.broadinstitute.org:3000/',
 ```
+Additionally, modify `cypress/support/commands.js` to point to this version of that url:
+```javascript
+  const url = `https://local.broadinstitute.org:3000`;
+```
```

Comment:
Perhaps we can figure a way to automate these actions, it can be confusing to a new engineer all the things they need to change in their environment before they can run tests. Perhaps we have a default url environment variable that is used across the stack and this uses that variable.